Patches/Alpha Animals/ThingDefs_Misc/AlphaAnimals_CE_Patch_Projectiles.xml

```diff
@@ -61,302 +59,301 @@
-					</projectile>
-				</value>
-			</li>
-
-			<li Class="PatchOperationReplace">
-				<xpath>/Defs/ThingDef[defName="AA_FireSpit"]/projectile</xpath>
-				<value>
-					<projectile Class="CombatExtended.ProjectilePropertiesCE">
-						<flyOverhead>false</flyOverhead>
-						<damageDef>Flame</damageDef>
-						<damageAmountBase>4</damageAmountBase>
-						<speed>15</speed>
-						<ai_IsIncendiary>true</ai_IsIncendiary>
-					</projectile>
-				</value>
-			</li>
-			
+						</comps>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="AA_FrostWeb"]/projectile</xpath>
+					<value>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<flyOverhead>false</flyOverhead>
+							<damageDef>Frostbite</damageDef>
+							<damageAmountBase>6</damageAmountBase>
+							<speed>15</speed>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+							<!-- Adding secondary damage with CE's Procedure instead of AA's AnimalProjectile -->
+							<secondaryDamage>
+								<li>
+									<def>Stun</def>
+									<amount>8</amount>
+								</li>
+							</secondaryDamage>
+						</projectile>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="AA_FireWeb"]/projectile</xpath>
+					<value>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<flyOverhead>false</flyOverhead>
+							<damageDef>Flame</damageDef>
+							<damageAmountBase>3</damageAmountBase>
+							<speed>15</speed>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+							<!-- Adding secondary damage with CE's Procedure instead of AA's AnimalProjectile -->
+							<secondaryDamage>
+								<li>
+									<def>Stun</def>
+									<amount>8</amount>
+								</li>
+							</secondaryDamage>
+						</projectile>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/ThingDef[defName="AA_AcidicWeb"]/projectile</xpath>
+					<value>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<flyOverhead>false</flyOverhead>
+							<damageDef>AA_AcidSpit</damageDef>
+							<damageAmountBase>6</damageAmountBase>
+							<speed>15</speed>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>
+							<!-- Adding secondary damage with CE's Procedure instead of AA's AnimalProjectile -->
+							<secondaryDamage>
+								<li>
+									<def>Stun</def>
+									<amount>8</amount>
+								</li>
+							</secondaryDamage>
+						</projectile>
+					</value>
+				</li>
+				<li Class="PatchOperationReplace">
+					<xpath>/Defs/VFECore.ExpandableProjectileDef[defName="AA_AcidicVomit"]/projectile</xpath>
+					<value>
+						<projectile Class="CombatExtended.ProjectilePropertiesCE">
+							<damageDef>AA_AcidSpit</damageDef>
+							<flyOverhead>false</flyOverhead>
+							<damageAmountBase>6</damageAmountBase>
+							<speed>20</speed>
+							<armorPenetrationBlunt>3</armorPenetrationBlunt>

```

Comment:
<damageAmountBase>12</damageAmountBase>
							<speed>8</speed>
							<armorPenetrationBlunt>1.5</armorPenetrationBlunt>

I would use these values, as an acidic spit is a more concentrated projectile, it should be faster and better-penetrating than just an unconcentrated stream of acid.

Reply:
Done.